### PR TITLE
New Navigation Bar on Header Menu and Drawer Menu for Mobiles

### PR DESCRIPTION
**Pull request by santisiri**
- Header menu now is lighter and adapted for better navigation
- Adapts automatically to all screen sizes
- All drop-downs and filtering menus also match the new look and fill
- Drawer menu on mobile also adapted to the new style
- A simpler footer with links to the DAO

This will make Proof of Humanity work properly across all kinds of devices regardless of screen size and with full functionality.

### CODE REVIEW

File: components/header.js [hunk 1]
```diff
@@ -171,9 +182,9 @@ function AccountSettingsPopup() {
 function MobileNavbar({ toggleMobileMenuOpen }) {
   const [accounts] = useWeb3("eth", "getAccounts");
 
-  const { t } = useTranslation();
+  // const { t } = useTranslation();
```
Comment (mmiguerodriguez) [inline components/header.js]: remove commented lines

File: components/header.js [hunk 2]
```diff
@@ -182,20 +193,37 @@ function MobileNavbar({ toggleMobileMenuOpen }) {
...
+                marginTop: "10px",
+                marginLeft: "80px",
+              }}
+              src="/images/democratic-poh-logo-white.svg"
+              height="auto"
+            />
+          </Link>
+        </Row>
+      </Col>
+      {/* {showSubmitProfile && (
```
Comment (mmiguerodriguez) [inline components/header.js]: same for commented lines

File: components/header.js [hunk 3]
```diff
@@ -296,39 +355,45 @@ export default function AppHeader() {
...
               {t("header_profiles")}
             </Link>
           </NextLink>
         </Row>
         <Row onClick={() => setMobileMenuOpen(false)}>
-          <MyProfileLink className="poh-drawer-text" />
+          <MyProfileLink className="poh-header-text poh-header-text-mobile poh-drawer-button" />
         </Row>
-        {accounts?.length === 0 && (
+        {/* accounts?.length === 0 && (
```
Comment (mmiguerodriguez) [inline components/header.js]: same for commented lines

**Review by mmiguerodriguez — APPROVED** [review]
LGTM, great work.
nit: remove long commented lines.
Overall changes are fine 👍 

For future PRs I'd expect for the deploy of the code changes to be done automatically before merging [☎️ @bilinkis, idk if vercel allows it, but we should have it]

File: components/header.js (hunk 1 above)
Comment (santisiri) [reply components/header.js]: lines removed 👍

**Comment by santisiri** [comment]
Removed all the commented lines @mmiguerodriguez 👍

**Pull request merged**